.github/workflows/greetings.yml

```diff
@@ -0,0 +1,22 @@
+name: Greetings
+
+on: [pull_request, issues]
+
+jobs:
+  greeting:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+    steps:
+    - uses: actions/first-interaction@v1
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        issue-message: |
+          Thank you for reporting your first issue! Be sure that we will be looking at it.
+          Please let the maintainers know if your issue has not got enough attention after a few days.
```

Comment:
I believe the message should be to thank the user for their contribution and explain the process so they know what to expect. In particular, we should let them know it's totally normal if they don't get an answer right away.

If we want to invite new contributors to reach out to maintainers directly (not sure we should) we'll have to explain how they're supposed to do it.